src/components/overview.jsx

```diff
@@ -7,35 +7,52 @@ class Overview extends React.Component {
     this.renderPlaygrounds();
   }
   renderPlaygrounds() {
-    let playgrounds = Array.prototype.slice.call(React.findDOMNode(this.refs.overview).getElementsByClassName('lang-playground'), 0);
-    for (let p in playgrounds) {
+    const playgrounds = Array.prototype.slice.call(React.findDOMNode(this.refs.overview).getElementsByClassName("lang-playground"), 0);
```

Comment:
I think this should be `ReactDOM.findDOMNode` since ecology got bumped to react 0.14